targets/nrf52.json

```diff
@@ -12,5 +12,5 @@
 		"src/device/nrf/nrf52.s"
 	],
 	"openocd-transport": "swd",
-	"openocd-target": "nrf51"
+	"openocd-target": "nrf52"
```

Comment:
This change is unrelated to the Nano 33 BLE: the Nano 33 BLE is a nrf52840, not a nrf52 (aka nrf52832). Yes, this is a bit confusing, but is also the convention used by Nordic Semiconductors.

Note that in my experience, the nrf51 OpenOCD target will work just fine for nrf52xxx series chips and I have intentionally used it here so that the board files are compatible with older versions of OpenOCD (this is one of those cases where comments in JSON would have been very useful: to explain that there is a good reason for this).

Reply:
Yes, OK, so I roll back this and the sister change and we can comment about this later, when #2214 or similar PR enables us have comments in target files.